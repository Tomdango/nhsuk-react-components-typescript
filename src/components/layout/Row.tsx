import React, { HTMLProps } from 'react';
import classNames from 'classnames';

const Row: React.FC<HTMLProps<HTMLDivElement>> = ({ className, ...rest }) => (
  <div className={classNames('nhsuk-grid-row', className)} {...rest} />
);

export default Row;
