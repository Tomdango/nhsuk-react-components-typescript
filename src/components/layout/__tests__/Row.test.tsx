import React from 'react';
import Row from '../Row';
import { shallow } from 'enzyme';

describe('Row', () => {
  it('matches snapshot', () => {
    const component = shallow(<Row />);
    expect(component).toMatchSnapshot('Row');
    component.unmount();
  });
});
