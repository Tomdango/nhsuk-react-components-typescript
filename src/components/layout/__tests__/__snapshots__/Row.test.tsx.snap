// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row matches snapshot: Row 1`] = `
<div
  className="nhsuk-grid-row"
/>
`;
